src/plonesocial/core/browser/tiles/newpostbox.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
+from plone import api
+from plone.tiles import Tile
+from plonesocial.core.browser.stream import StreamBase
+
+
+class NewPostBoxTile(Tile, StreamBase):
+
+    index = ViewPageTemplateFile('templates/new-post-box-tile.pt')
+
+    def render(self):
+        return self.index()
+
+    def __call__(self):
+        if self.request.form:
+            view = api.content.get_view(
+                'stream',
```

Comment:
Why is there a connection between the post box and the stream view? Those are separate functionalities in separate tiles, right?